app/services/users/services.rb

```diff
@@ -18,7 +18,7 @@ def initialize(user)
     #
     def authorizations
       @list << BackendServices::RX if user.authorize :mhv_prescriptions, :access?
-      @list << BackendServices::MESSAGING if user.authorize messaging_service, :access?
+      @list << BackendServices::MESSAGING if user.authorize :mhv_messaging, :access?
```

Comment:
❓ The associated policy looks like it'll make an HTTP request each time to determine if the user has access. Do you have an idea of how long the request takes to complete?

Reply:
@dickdavis This is a valid concern, actually it was something I had brought up earlier (see my comments in: https://github.com/department-of-veterans-affairs/vets-api/pull/15889). However as far as I know those concerns were already accommodated, or at least are being watched. 

This PR is just cleaning up a feature flag that's already enabled as far as I can tell, basically the HTTP request is already happening

Reply:
@dickdavis my understanding is that `MHVSessionBasedClient` should be handling that and using data from cache if a previous request has been made and is not yet expired